flutter_local_notifications/ios/Classes/FlutterLocalNotificationsPlugin.m

```diff
@@ -113,6 +116,16 @@ - (void)pendingNotificationRequests:(FlutterResult _Nonnull)result {
                 if (request.content.userInfo[PAYLOAD] != [NSNull null]) {
                     pendingNotificationRequest[PAYLOAD] = request.content.userInfo[PAYLOAD];
                 }
+                if (request.content.attachments != nil) {
```

Comment:
Will be removing this as it's not actually being read from the Flutter/Dart side. It's also iOS specific